jekyll/_cci2/aws.adoc

```diff
@@ -0,0 +1,232 @@
+[#install]
+= Installation on AWS with Terraform
+:page-layout: classic-docs
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
+
+Following is a step by step guide to installing CircleCI Server v2.17 with Terraform.
+
+toc::[]
+
+== Define Variables for Terraform
+
+. Clone the https://github.com/circleci/enterprise-setup[Setup] repository. If you already have it cloned, make sure it is up-to-date and you are on the `master` branch by running:
++
+```shell
+git checkout master && git pull
+```
+. Go to the top directory of the `enterprise-setup` repo on your local machine.
+
+. Run `terraform init` to initialize your working directory.
+
+. Run `make init` to initialize a `terraform.tfvars` file (your previous `terraform.tfvars` if any, will be backed up in the same directory).
+
+. Open `terraform.tfvars` in an editor and fill in appropriate AWS values for section 1.
+
+. If you plan to use 1.0 builders, specify a `circle_secret_passphrase` in section 2, replacing `...` with alpha numeric characters, if not, leave it as is. 1.0 builders are disabled by default in section 3.
+
+. Specify the instance type to use for your Nomad clients. By default, the value specified in the `terraform.tfvars` file for Nomad Clients is `m4.2xlarge` (8 vCPUs, 32GB RAM). To increase the number of concurrent CircleCI jobs that each Nomad Client can run, modify section 2 of the `terraform.tfvars` file to specify a larger `nomad_client_instance_type`. Refer to the AWS https://aws.amazon.com/ec2/instance-types[Amazon EC2 Instance Types] guide for details.
++
+NOTE: The `builder_instance_type` is only used for CircleCI 1.0 and is disabled by default in section 3.
+
+. In section 3 you can:
+.. choose to use 1.0 Builders if your project requires it (by changing the count to `1`)
```

Comment:
I wonder if we should get rid of this in the documentation? This seems like something we should not even suggest at this point (using 1.0 builders)

Reply:
We should ask George to get this out of the UI I think.

Reply:
Yes good idea, it is a bit confusing what's going on with this in Replicated and the UI: https://docs.google.com/document/d/1G_rk0LJzS_8qY1xxlBA5ROZwoaoH7azY5MIyU297YIg/edit